modules/configuring-access-control-and-sending-invitations.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * configuration/inviting-users-to-rhacs.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-access-control-and-sending-invitations_{context}"]
+= Configuring access control and sending invitations
+
+.Procedure
+
+. To invite a user, use one of the following methods:
+* In the {product-title-short} portal, navigate to *Platform Configuration -> Access Control -> Auth providers* tab, and then click *Invite users*.
+* Navigate to your profile icon, and then click *Invite users*.
+
+. In the *Invite users* dialog box, provide the following information:
+* *Emails to invite*: Enter one or more email addresses of the users you want to invite. Ensure that they are valid email addresses associated with the intended recipients.
+* *Provider*: From the drop-down list, select a provider you want to use for each invited user. For {product-title-managed-short}, the default option is `Default Internal SSO`.
+* *Role*: From the drop-down list, select a role for each invited user.
+
+. Click *Invite users*.
+. On the *Invite users* dialog box, click *Done.*
+. On the confirmation dialog box, click *Done.*
```

Comment:
There is no second confirmation box; only one more screen after you click the Invite users button to take the action.

Also, there is no ability for ACS to send this type of transactional email at this time, so there needs to be a step before clicking Done, where the inviter copies the email address and a bolierplate message into their own email client to send to the invited users.

```suggestion
. When the confirmation that the users have been created with the selected role, copy the emails and the message into an email that you create in your own email client to send to the users.
. Click *Done.*
```